Source/Core/Core.swift

```diff
@@ -1012,7 +1012,10 @@ extension FormViewController {
         let endFrame = keyBoardInfo[UIResponder.keyboardFrameEndUserInfoKey] as! NSValue
 
         let keyBoardFrame = table.window!.convert(endFrame.cgRectValue, to: table.superview)
-        let newBottomInset = table.frame.origin.y + table.frame.size.height - keyBoardFrame.origin.y + rowKeyboardSpacing
+        var newBottomInset = table.frame.origin.y + table.frame.size.height - keyBoardFrame.origin.y + rowKeyboardSpacing
+        if #available(iOSApplicationExtension 11.0, *) {
```

Comment:
Why not `iOS` instead of `iOSApplicationExtension`?

Reply:
It's weird. `iOSApplicationExtension` is auto fixed by Xcode. Code has been updated with `#available(iOS 11.0, *)` now.